tests/lightning_qubit2/test_expval_2.py

```diff
@@ -0,0 +1,340 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for process and execute (expval calculation).
+"""
+# pylint: disable=too-many-arguments, redefined-outer-name
+import numpy as np
+import pennylane as qml
+import pytest
+from conftest import LightningDevice, THETA, PHI, VARPHI
+from pennylane.devices import DefaultQubit
+
+from pennylane_lightning.lightning_qubit import LightningQubit2
```

Comment:
The idea of importing `LightningQubit2`, in such cases, is to check if `LightningDevice` is this device.
We can, but we shouldn't have tests in terms of `LightningQubit2`, because they are not easily generalized to other devices in the future.

Reply:
Good point. A more appropriate check would be to see if `LightningDevice._new_API`, which would generalize to other devices once ported to the new device API.